packages/create-amplify/src/get_project_root.ts

```diff
@@ -1,13 +1,20 @@
 import fsp from 'fs/promises';
 import path from 'path';
+import yargs from 'yargs';
 import { AmplifyPrompter, LogLevel } from '@aws-amplify/cli-core';
 import { printer } from './printer.js';
 
 /**
  * Returns the project root directory.
  */
 export const getProjectRoot = async () => {
-  const useDefault = process.env.npm_config_yes === 'true';
+  const argv = await yargs(process.argv.slice(2)).options({
```

Comment:
I'd rather not.
This will break this: https://github.com/aws-amplify/amplify-backend/blob/348717b55121d63560524378588e52184ed2f95b/packages/integration-tests/src/test-live-dependency-health-checks/health_checks.test.ts#L45
And. we'll loose `--help` output :(

Reply:
having said that, we should probably refactor how do we use yargs in this package.

Reply:
btw. that regression happened on `main` so if we want to fix it it can be a follow up.

Reply:
Can't reproduce the regression on amplify@0.6.0, which is this PR ✌️

Reply:
I'm good with cutting a ticket to track and doing this in a follow up

Reply:
Issue to remove `yarngs` in a follow-up PR https://github.com/aws-amplify/amplify-backend/issues/972